### PR TITLE
Fixes AMD container recipes

### PR DESCRIPTION
**Pull request by dipietrantonio**
Dear @vsoch, by mistake in the last pull request I ended up including old versions of the recipes and not the ones I had tested. Apologies for that. Here are the ones without weird blanks and typos.

Thank you again for your patience.

Cristian

### CODE REVIEW

File: amd/infinityhub/cp2k/container.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+docker: amdih/cp2k
```
Comment (vsoch) [inline amd/infinityhub/cp2k/container.yaml:1]: The docker URI here (and for the others below) should match the path of the recipe, so this should be under `amdih/cp2k`

Comment (dipietrantonio) [reply amd/infinityhub/cp2k/container.yaml]: @vsoch thanks for the feedback, I made the change.

File: amdih/cp2k/container.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+docker: amdih/cp2k
+url: https://www.amd.com/en/technologies/infinity-hub/cp2k
+description: CP2K is a quantum chemistry and solid state physics software package that can perform atomistic simulations of solid state, liquid, molecular, periodic, material, crystal, and biological systems. 
+maintainer: "@cristiandipietrantonio"
+latest:
+  '8.2': sha256:5947603de32c4e690f734075f7625563a2de18d1c3276ec10fceccdb332022ec
+tags:
+  '8.2': sha256:5947603de32c4e690f734075f7625563a2de18d1c3276ec10fceccdb332022ec
+aliases:
+- name: cp2k.popt
```
Comment (vsoch) [inline amdih/cp2k/container.yaml:10]: You just fixed the path as I was writing a comment - very speedy!

Another question - is there any reason you aren't putting full paths? We typically put full paths to the executables (aliases).

Comment (dipietrantonio) [reply amdih/cp2k/container.yaml:10]: Hi @vsoch, no particular reason. I will update the recipes when I have a bit of time, thanks for the suggestion.

**Review by vsoch — APPROVED** [review]

**Pull request merged**